src/hdmf/container.py

```diff
@@ -302,6 +303,38 @@ def get_ancestor(self, **kwargs):
             p = p.parent
         return None
 
+    def all_children(self):
+        """
+        This method will search through all children within the object and their children.
```

Comment:
```suggestion
        """Get a list of all child objects and their child objects recursively.
```

Reply:
In general, the first line of a method/function docstring should be a succinct description of what the method/function does / what it returns